thriftbp/server.go

```diff
@@ -106,12 +124,7 @@ func NewBaseplateServer(
 	)
 	middlewares = append(middlewares, cfg.Middlewares...)
 	cfg.Middlewares = middlewares
-	cfg.Logger = log.ZapWrapper(log.ZapWrapperArgs{
-		Level: bp.GetConfig().Log.Level,
-		KVPairs: map[string]interface{}{
-			"from": "thrift",
-		},
-	}).ToThriftLogger()
+	cfg.Logger = buildTServerLogger(bp.GetConfig().Log.Level, cfg.ThriftSocketTimeout > 0)
```

Comment:
🔕 Can this be rewritten to be like

```
cfg.Logger = ...
if cfg.ThriftSocketTimeout > 0 {
  cfg.Logger = suppressTimeoutLogger(cfg.Logger)
}
```

?

Reply:
fixed